Updating description fields (#6852)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/custom-aliases.md

---
title: "Custom aliases"

> description: "Learn how to use custom aliases when building in dbt."

Author: ```suggestion
description: "Configure custom aliases to override default naming conventions in dbt."
```

> description: "Configure custom aliases to override default naming conventions in dbt."

Reviewer: ```suggestion
description: "Configure custom aliases to override default model naming conventions in dbt."
```

Reviewer 2: this reads great! you can use aliases for models, but other resources as well (seeds, snapshots, etc) so i'd enhance Leona's sugg as:

> description: "Override default naming conventions in dbt by configuring custom aliases for models, tests, and other resources"

> description: "Configure custom aliases to override default naming conventions in dbt."

Author: ```suggestion
description: "Configure custom aliases to override default naming conventions for models and other resources in dbt."
```
